src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/StomperBoots.java

```diff
@@ -89,4 +91,20 @@ private Vector getShockwave(Location origin, Location target) {
         }
     }
 
+    /**
+     * Checks if the stomper boots can move an entity and is not the player who is using the boots.
+     * <br><b>For developers:</b> If you're spawning an immovable NPC, you should be denying
```

Comment:
According to the javadocs convention `<br>` should not be used. Instead `<p>` should be used to start a new paragraph.
Although you will see the line break quite often in this codebase already because I didn't know that until recently either.

Reply:
Was doing it for consistency but sute